notebooks/attack_laser.ipynb

```diff
@@ -0,0 +1,678 @@
+{
```

Comment:
Can we make the first cell for path definition generic?

Reply:
Please add title at the very top and texts providing explanations for the main steps and obtained results.